src/Controls/src/Core/Compatibility/Handlers/TableView/Android/TableViewRenderer.cs

```diff
@@ -107,7 +137,10 @@ public override SizeRequest GetDesiredSize(double widthConstraint, double height
 							continue;
 						}
 
-						AView listItem = _adapter.GetView(i, null, Control);
+						var platformView = cell.Handler?.PlatformView as AView;
+						var convertView = (platformView?.Parent as AView) ?? platformView;
+
+						AView listItem = _adapter.GetView(i, convertView, Control);
```

Comment:
Could this be null?, because the PlatformView and the Parent is null.

Reply:
`listItem` won't be null but the convertView might. Which is fine if this is the first time the handler is being realized.